src/utils/property-filters/getMainPagePropertyFilter.ts

```diff
@@ -22,6 +34,13 @@ const getMainPagePropertyFilter = (formData: PropertyFilterFormData): string =>
         district: location?.searchParams.district ?? '',
       })
     );
+  if (rent === 'for_sale') {
```

Comment:
We have a enum for that.  
Check QobrixPropertySaleRent enum

Reply:
fixed